src/js/content_scripts/injector.js

```diff
@@ -93,7 +93,22 @@ const makeButton = (text, callback, color='rgb(0, 153, 255)', icon='') => {
   `;
 };
 
-function loaded() {
+const constructParams = () => {
+  const params = new URLSearchParams(window.location.search);
+  params.set(
+    'video',
+    (
+      params.get('v') ||
+      (new URLSearchParams(window.parent.location.search).get('v'))
+    )
+  );
```

Comment:
Please clean up by moving the parent parameters stuff to a separate statement so you can make the ``params.set`` into one line.